src/open_inwoner/cms/cases/views/status.py

```diff
@@ -222,6 +222,13 @@ def get_context_data(self, **kwargs):
             hooks.case_status_seen(self.request.user, self.case)
             hooks.case_documents_seen(self.request.user, self.case)
 
+            # used in template + for disabling document upload message
+            self.has_recently_uploaded_documents = has_new_elements(
+                documents,
+                "created",
+                dt.timedelta(days=settings.DOCUMENT_RECENT_DAYS),
```

Comment:
@alextreme I see this is now a configurable number of days, would it be possible to instead check if there are new documents since the current status was added? Or could there be a scenario where a user has to upload documents at different points in the time while the same zaakstatus is active?

Reply:
@stevenbal that is an option, however a case could be in the same state for a few weeks or quickly toggle between multiple statusses. We discussed a few alternatives but this one was the easiest and avoids surprises